### PR TITLE
Chore: growthbook

### PR DESCRIPTION
**Pull request by dleard**

### CODE REVIEW

**Review by mikevespi — APPROVED** [review]
Looks great!

**Review by mikevespi — APPROVED** [review]
:shipit:

File: app/pages/_app.tsx [hunk 1]
```diff
@@ -27,6 +32,24 @@ function MyApp({ Component, pageProps }: AppProps) {
...
+  useMemo(() => {
+    const fetchGrowthbook = async () => {
+      try {
+        const data = await fetch(
+          `https://cdn.growthbook.io/api/features/${publicRuntimeConfig.GROWTHBOOK_API_KEY}`
+        )
+          .then((res) => res.json())
+          .then((res) => {
+            growthbook.setFeatures(res.features);
+          });
```
Comment (pbastia) [inline app/pages/_app.tsx:44]: this is not a response object anymore
```suggestion
          .then((jsonData) => {
            growthbook.setFeatures(jsonData.features);
          });
```

File: app/pages/_app.tsx [hunk 2]
```diff
@@ -27,6 +32,24 @@ function MyApp({ Component, pageProps }: AppProps) {
     setError(null);
   }, [Component]);
 
+  useMemo(() => {
```
Comment (pbastia) [inline app/pages/_app.tsx:35]: I would add a small comment about the caveat that the use of `useMemo` here means the feature toggle in growthbook means the change won't be visible for a user until they refresh the page

File: docs/developer-guide.md [hunk 3]
```diff
@@ -3,3 +3,46 @@
...
+
+In order for the feature toggling in the Growthbook dashboard to actually do anything, we need to add some code to wrap anything we want to be able to show/hide.
+
+Example:
+
+```typescript
+import { useFeature } from "@growthbook/growthbook-react"; // Import the useFeature() function
+...
+...
+const showFlaggedFeature = useFeature('flagged-item').on; // Set the value of useFeature.on to a boolean variable
```
Comment (pbastia) [inline docs/developer-guide.md:40]:
```suggestion
const showFlaggedFeature = useFeature('flagged-feature').on; // Set the value of useFeature.on to a boolean variable
```

**Review by pbastia — CHANGES_REQUESTED** [review]
minor code style, awesome 👌

**Review by pbastia — APPROVED** [review]

**Pull request merged**